src/HeaderCol.res

```diff
@@ -1,10 +1,24 @@
+open ReactUtils
+
 type colType = Id | Description | Source | Target
 
-let getColType = (index, canToggleDescription) => {
+// This never got the correct result for me
+// "canToggleDescription" was always false and Source + Target Row were both interpreted as Target Rows
```

Comment:
does it work when you make the initial state of mode `Json` instead of `Other`? Seems to be a bug that it is not set in all cases (e.g. when the app is loaded from storage)

Reply:
Yes, that works! :-)